autogpt_platform/frontend/src/components/CustomNode.tsx

```diff
@@ -241,6 +243,55 @@ export const CustomNode = React.memo(
       return renderHandles(schema.properties);
     };
 
+    const generateAyrshareSSOHandles = (
```

Comment:
`CustomNode`, if I'm correct, is used on all blocks. Are we certain we want the Ayrshare login to appear on all blocks potentially? 👀